sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/BarrierRequestHelper.java

```diff
@@ -3,7 +3,7 @@
 
 package com.azure.cosmos.implementation.directconnectivity;
 
```

Comment:
Please consider sorting the imports here as one of them may now be out of order.